lib/util/propTypes.js

```diff
@@ -12,6 +12,20 @@ const variableUtil = require('./variable');
 const versionUtil = require('./version');
 const propWrapperUtil = require('./propWrapper');
 const getKeyValue = require('./ast').getKeyValue;
+const findReturnStatement = require('./ast').findReturnStatement;
+
+/**
+ * Check if node is function type.
+ * @param {ASTNode} node
+ * @returns {Boolean}
+ */
+function isFunctionType(node) {
+  if (node.type) {
+    const regexp = new RegExp('^(?:Function(?:Declaration|Expression)|ArrowFunctionExpression)$');
```

Comment:
```suggestion
    const regexp = /^(?:Function(?:Declaration|Expression)|ArrowFunctionExpression)$/;
```
`new RegExp` should never be used when a literal suffices.

altho this seems like it'd be clearer as a chain of `if`s rather than using a regex.

Reply:
Thanks for your advice!
How about using Array.includes ?
```
function isFunctionType(node) {
  if (node.type) {
    return ['FunctionDeclaration', 'FunctionExpression', 'ArrowFunctionExpression'].includes(node.type);
  }
  return false;
}
```

Reply:
That also seems like needless runtime overhead for only three choices. I'd prefer a simple if chain.